src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -47,18 +47,7 @@ export default function PreferencesComponent() {
 				successMessage='updated.preferences'
 				failureMessage='failed.to.submit.changes'
 			/>
-			<div style={bottomPaddingStyle}>
-				<p style={titleStyle}>
-					{`${translate('default.site.title')}:`}
-				</p>
-				<Input
-					type='text'
-					placeholder={translate('name')}
-					value={localAdminPref.displayTitle}
-					onChange={e => makeLocalChanges('displayTitle', e.target.value)}
-					maxLength={50}
-				/>
-			</div>
+			<h3 className='border-bottom'>Graph Settings</h3>
```

Comment:
This and the other two section headers are not translated.

Could this entire section be put last on the page? It is the least commonly used so it seems best last (as were many of these items before).

Reply:
Fixed in [e74a45d](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/e74a45dfd49245ab71551a31a40010a1f144186f)